tests/playTests.sh

```diff
@@ -485,6 +485,11 @@ $ZSTD -bi0 --fast tmp1
 $ECHO "with recursive and quiet modes"
 $ZSTD -rqi1b1e2 tmp1
 
+$ECHO "\n===>  zstd compatibility tests "
+
+./datagen > tmp
+$ZSTD --format=zstd tmp 2> tmplog
+grep "zst" tmplog > $INTOVOID || die "--format=zstd not supported"
```

Comment:
This test works.
Just, if all you want to do is to check that a file named `tmp.zst` was created,
the way we do it in other equivalent tests in this same script is : 
`test -f tmp.zst`

If you want to be sure that the file `tmp.zst` was actually created by your test, and not just already present due to some earlier artefact, `rm -f tmp.zst` before compression.